src/main/java/games/strategy/engine/vault/Vault.java

```diff
@@ -345,13 +345,13 @@ public void waitForIdToUnlock(final VaultID id, final long timeout) {
       }
     }
   }
-}
-
 
-interface IRemoteVault extends IChannelSubscribor {
-  void addLockedValue(VaultID id, byte[] data);
+  static interface IRemoteVault extends IChannelSubscribor {
```

Comment:
`static` is implied on nested interfaces and is redundant.

Another RMI use case here.  Just wanted to confirm there are no issues with moving an `IChannelSubscribor`.  I think based on our experience testing the lobby a few weeks ago, the only thing that matters here are the method names and parameters, correct?